src/main/cluster.ts

```diff
@@ -48,10 +48,31 @@ export interface ClusterState {
   allowedResources: string[]
 }
 
+/**
+ * Cluster
+ *
+ * @beta
```

Comment:
This PR should probably have some doc describing what we mean by these markers and what guarantees (or not) they mean.